internal/project/framework/internals/error/error.go

```diff
@@ -7,7 +7,8 @@ import (
 	"fmt"
 	"io"
 
-	"{{ .PkgPath }}/framework/internals/stacktrace"
+	/**#bean*/
+	"demo/framework/internals/stacktrace" /*#bean.replace("{{ .PkgPath }}/framework/internals/stacktrace")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines